eng/Subsets.props

```diff
@@ -267,6 +267,22 @@
       Category="clr" />
   </ItemGroup>
 
+  <!-- Build the CoreCLR cross tools when we're doing a cross OS build for apple device targets -->
```

Comment:
Could this be folded into the above `Build the CoreCLR cross tools when we're doing a cross build` block? We currently exclude it with `'$(TargetsMobile)' != 'true'` but it looks pretty much the same. The additional `TargetOS` and `OutputRIDOS` properties will probably not harm anything if we add them.

Reply:
I think so. Initially, I didn't want to extend the cross-build for unsupported scenarios.